Tests/DocumentationTests/RegexBuilderTests.swift

```diff
@@ -20,6 +20,7 @@ import RegexBuilder
 class RegexBuilderTests: XCTestCase {}
 
 extension RegexBuilderTests {
+  @available(macOS 13, *)
```

Comment:
I've never had availability like this limit where tests run. I always have to do a runtime check inside the method itself, at least on macOS / Xcode. I'd guess the Obj-C dynamic lookup doesn't see the availability, so it tries to run the test anyway. Is there something special about these tests?

Also, why put all methods in an extension?

Reply:
Yeah, I thought this would work, but no… 😞

Re: extension, I think I'm cargo culting the test classes in the other test targets.